.github/workflows/push-master.yaml

```diff
@@ -13,12 +13,11 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: 16.15.0
-    - run: npm ci --legacy-peer-deps
-    - run: npm run build --if-present
-    - run: npm test
+    - run: yarn install --immutable
+    - run: yarn test
     - name: Publish to npm
       run: |
-        echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
-        npm publish
+        echo 'npmRegistryServer: "https://registry.npmjs.org/"\nnpmAuthToken: "$NPM_TOKEN"' > ./.yarnrc
```

Comment:
So [is this broken too](https://github.com/smartprocure/contexture/blob/master/.github/workflows/push-master.yaml#L26)?

Reply:
Yeah https://github.com/smartprocure/contexture/actions/runs/3314424542/jobs/5473698231